controllers/config_helper.go

```diff
@@ -0,0 +1,249 @@
+package controllers
+
+import (
+	"context"
+
+	operatorv1alpha1 "github.com/gatekeeper/gatekeeper-operator/api/v1alpha1"
+	"github.com/go-logr/logr"
+	"github.com/open-policy-agent/gatekeeper/v3/apis/config/v1alpha1"
+	"github.com/open-policy-agent/gatekeeper/v3/pkg/wildcard"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	cacheRuntime "sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/metrics/server"
+)
+
+// Default config data
+func getDefaultConfig(namespace string) *v1alpha1.Config {
+	config := &v1alpha1.Config{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: namespace,
+			Name:      "config",
+		},
+		Spec: v1alpha1.ConfigSpec{
+			Match: []v1alpha1.MatchEntry{
+				{
+					ExcludedNamespaces: []wildcard.Wildcard{
+						"kube-*", "multicluster-engine",
+						"hypershift", "hive", "rhacs-operator", "open-cluster-*",
+					},
+					Processes: []string{
+						"audit", "webhook", "sync",
```

Comment:
Why did you choose this instead of `*`? Also, I'm starting to think that we shouldn't prevent the user from auditing. The main objective I think is to prevent the webhook from bricking the cluster.

@gparvin what do you think?

Reply:
Oh good!! so should we add only `webhook` and `sync`

Reply:
also should we add "openshift-*"?

Reply:
That's what I'm thinking but would like @gparvin's opinion on this.